test/coverage.yaml

```diff
@@ -34,8 +34,10 @@ directories:
   source/extensions/common/tap: 94.6
   source/extensions/common/wasm: 95.3  # flaky: be careful adjusting
   source/extensions/common/wasm/ext: 92.0
+  source/extensions/filters/common: 95.3
   source/extensions/filters/common/fault: 94.5
   source/extensions/filters/common/rbac: 92.6
+  source/extensions/filters/common/lua: 87.2
```

Comment:
That's not great; any way to increase coverage?

Reply:
You'd need to define a protobuf message with all those types in it, and set it in metadata and get it from lua.

Reply:
@ggreenway Thanks! I was able to bring it up to 92.xx%

Reply:
@ggreenway Does this look okay now? I can resolve the merge conflict and make it ready if there is no other feedback.